runtime/src/main/cpp/ObjCExportErrors.mm

```diff
@@ -19,6 +19,7 @@
 #import <Foundation/NSDictionary.h>
 #import <Foundation/NSError.h>
 #import <Foundation/NSString.h>
+#import <stdio.h>
```

Comment:
Hmm, why not rely on konan::consolePrintf, adding new logging mechanism is really not desirable.

Reply:
Fixed.